packages/kolibri-common/components/SearchBox.vue

```diff
@@ -117,6 +124,12 @@
           textAlign: this.isRtl ? 'right' : '',
         };
       },
+      computedPlaceholder() {
+        if (this.windowBreakpoint <= 3.5) {
```

Comment:
We should just test for 3 and 4 breakpoints, so lets update this validation to `if ([3, 4].includes(this.windowBreakpoint)) {`